### ISSUE TITLE
Issues with Biset Tracker

### ISSUE THREAD

**Issue by MusicByJethro** [DISTRHO/Cardinal#686]
### Version

24.05

### Plugin type

VST3

### OS

MS11

### DAW / Host (if applicable)

FL Studio

### Description

Biset Tracker randomly stops playing patterns, and will not play them. I cant figure out if this is a Biset issue specifically, or if this is just an issue with the port into Cardinal. And when I say that it stops playing the pattern, it's in its entirety, No CV output, and no gate output. and it seems to happen at random, and I cant replicate it faithfully.  I know that that makes this entry less than helpful, but i figured the best i could do is make someone aware that there are some issues still to be worked out. As far as I can tell, its entire synth channels within the tracker that will seemingly just stop working, so it isn't exclusive to just a pattern. I will update if i can figure out what's causing this, or if at least i can replicate the issue.

**Comment by falkTX** [comment]
maybe you are adding more than 1 instance? Biset modules can only be loaded once, if there are multiple cardinal plugins with biset on them it is unclear what happens to Biset side

**Comment by MusicByJethro** [comment]
> maybe you are adding more than 1 instance? Biset modules can only be loaded once, if there are multiple cardinal plugins with biset on them it is unclear what happens to Biset side

I dont believe this was the case. I'm fairly anal when it comes to instances of things being up, and have since encountered the same bug. its just frustrating because it seems to happen very randomly, and basically ruins progress on a patch. are there any other sequencers that can do arrangements? I feel like Biset makes the only one we have available in Cardinal.
